vidarr-cromwell/README.md

```diff
@@ -34,6 +35,16 @@ with two arguments, the file to copy (which will be provided as the argument
 collect the permanent location for the file (`"storagePathField"`), the MD5
 (`"md5Field"), and the file size ("fileSizeField").
 
+Your file system probably will not appreciate having thousands of files dumped
+in a single output directory, so the `"chunks"` parameter will create a
+heirarchy of directories based on the workflow run identifier. The numbers
```

Comment:
heirarchy -> hierarchy

Reply:
I can't spell this word to save my life and it comes up a lot in computing.